docs/Erlang Distribution.md

```diff
@@ -0,0 +1,92 @@
+# Erlang Distribution
+
+## Static Configuration
+
+The simplest way to configure Erlang distribution is to statically (i.e., at
+compile time) set the node name and cookie. This is done by editing the
+`rel/vm.args` file, which contains the command-line arguments used to launch
+the Erlang/OTP virtual machine.
+
+You would simply add `-name` and `-setcookie` parameters, as follows:
```

Comment:
I add `nerves.local` (and similar names) in my laptop's `/etc/hosts`, which enables me to connect to the Nerves nodes using `iex`.